### ISSUE TITLE
Redump parsing issue

### ISSUE THREAD

**Issue by alucryd** [SnowflakePowered/shiratsu#10]
Hi there, I've been parsing redump DATs without issues for a while, but I just stumbled upon 2 errors, one with the PlayStation 3 DAT, and another with the Neo Geo CD DAT.

```
{ err: "Failed to parse no-intro name, The name \"Call of Duty - Advanced Warfare (United Arab Emirates) (En,Ar)\" could not be parsed properly in the NoIntro naming convention" }
{ err: "Failed to parse no-intro name, The name \"Double Dragon (Export) (En,Ja)\" could not be parsed properly in the NoIntro naming convention" }
```

Is there something you can/should do on your end, or should I give up on these?

**Comment by chyyran** [comment]
I think this is just an issue with country names. I'll push a quick fix later today.

**Comment by chyyran** [comment]
Fixed in https://github.com/SnowflakePowered/shiratsu/commit/85a5d1134adb10a44521913c30050ce107305979

published as `shiratsu-naming = "0.1.5"`

**Comment by alucryd** [comment]
Thanks for the quick fix!
